.gitignore

```diff
@@ -1,3 +1,4 @@
 **/web-ext-artifacts
 **/*.xpi
 node_modules
+.idea/
```

Comment:
Ah can you remove this? If this is an issue it should probably rather go into your ~/.gitignore_global

Reply:
Not really an issue, I'm reverting that!

Reply:
And thanks for the gitignore_global tip